packages/babel-parser/src/plugins/flow.js

```diff
@@ -1932,36 +1961,16 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       for (let i = 0; i < exprList.length; i++) {
```

Comment:
Nit: we can use for/of here, since we compile it using the `assuneArray` option (`for (a of b)` -> `for (let i = 0; i < b.length ..`)

Reply:
I leave that for now, i didn't want to make the commit diff bigger. Can do that in a separate PR.